vllm/core/block/cpu_gpu_block_allocator.py

```diff
@@ -190,10 +190,16 @@ def clear_copy_on_writes(self) -> Dict[int, List[int]]:
         device = Device.GPU
         return self._allocators[device].clear_copy_on_writes()
 
-    def mark_blocks_as_computed(self) -> None:
+    def mark_blocks_as_accessed(self, block_ids: List[int],
+                                now: float) -> None:
         # Prefix caching only supported on GPU.
         device = Device.GPU
-        return self._allocators[device].mark_blocks_as_computed()
+        return self._allocators[device].mark_blocks_as_accessed(block_ids, now)
```

Comment:
can we validate that the block ids are on GPU? otherwise the error message will be pretty confusing

Reply:
In prefix_caching_block.py, I track the already cached blocks in _blocks. only mark those id if they are in the cached pool, I think it shall be enough for validation of GPU.

Reply:
I see, I would raise error when found the block is not belong to gpu vllm/core/block/prefix_caching_block.py.